### PR TITLE
[SEMI-MODULAR]Fixes playtime cloak & adds support for admins

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
Does as the title says.
However; to now clarify

-The 'Veteran' status has been reduced to 2000 hours from 5000
-The sprite has been majorly overhauled with a variety of more frames and colors, now at 20fps!
-It no longer looks terrible
-Description and name reworked
-R_ADMIN bypass

Major props to whoever made the sprite this was based off  of. Lovely work!

## Why It's Good For The Game
This didn't work.

## Changelog
:cl:
fix: Veteran Cloak actually works!
fix: Veteran cloak should (now) stop the unenlightened from wearing it.
add: '''New''' sprite states for it, making it look much less epileptic.
/:cl:

### CODE REVIEW

**Comment by Vizoplez** [comment]
Ignore the strange change in weapon_examine, its because this repo was sliiightly outdated.

Master is up to date anyway with that change, no diff

**Comment by Vizoplez** [comment]
also don't merge yet correcting an issue.

**Comment by Vizoplez** [comment]
okay it no longer seizures.

File: code/__DEFINES/misc.dm [hunk 1]
```diff
@@ -526,7 +526,8 @@ GLOBAL_LIST_INIT(pda_styles, sortList(list(MONO, VT, ORBITRON, SHARE)))
 
 // Play time / EXP
 #define PLAYTIME_HARDCORE_RANDOM 120
-#define PLAYTIME_VETERAN 300000 //Playtime is tracked in minutes. 300,000 minutes = 5,000 hours
+#define PLAYTIME_VETERAN 120000 // SKYRAT EDIT - Playtime is tracked in minutes. 120,000 minutes = 2000 hours //Almost nobody has this, and with us adding new jobs and changing new ones, this number fluctuates. Alot.
+
```
Comment (KathrinBailey) [inline code/__DEFINES/misc.dm:530]: sKyRaT eDiT

Comment (Vizoplez) [reply code/__DEFINES/misc.dm:530]: ?

File: code/datums/elements/weapon_description.dm [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#define HITS_TO_CRIT(damage) round(100 / damage, 0.1)
+#define HITS_TO_CRIT(damage) round(130 / damage, 0.1) //Skyrat Edit
```
Comment (tf-4) [inline code/datums/elements/weapon_description.dm:1]: ?

**Comment by SpaceVampire** [comment]
You know, I'm pretty sure you made it so that admins can pick the cloak in their load-out.
But they still won't be able to actually wear it unless they have the hours.
Thus making that kind of pointless.
But anyway, I think the cloak doesn't look very good.
You should maybe put an image of it in the actual PR.
It also has the same issue behind department skill-capes.
See the earlier conversation about why we didn't want to add em.
Simi-OOC bragging rights are really kinda iffy.

File: code/datums/elements/weapon_description.dm (hunk 2 above)
Comment (KathrinBailey) [reply code/datums/elements/weapon_description.dm:1]: uwu what's this?
Seriously though what's this for?

**Review by KathrinBailey — CHANGES_REQUESTED** [review]
Honestly not sure if we should give admins the cloak, it is specifically for high playtime. This kinda dilutes the significance of the cloak.

**Comment by tf-4** [comment]
Pretty much what SpaceVampire said. It doesn't look great, and OOC rewards IC are very questionable.

**Comment by Gandalf2k15** [comment]
Not a fix tho

**Pull request closed without merging**